FinniversKit/Sources/Recycling/GridViews/NeighborhoodProfile/NeighborhoodProfileView.swift

```diff
@@ -20,7 +20,30 @@ public final class NeighborhoodProfileView: UIView {
 
     // MARK: - Private properties
 
-    private var viewModel = NeighborhoodProfileViewModel(title: "", readMoreLink: nil, cards: [])
+    private var viewModel = NeighborhoodProfileViewModel(title: "", readMoreLink: nil, cards: [], banner: nil)
```

Comment:
Since we got rid of `systemLayoutSizeFitting` this property can become an optional instead. It's a bit confusing why we initialize it with default values that aren't used at all.

```swift
private var viewModel: NeighborhoodProfileViewModel?
```

Reply:
By changing it to an optional, we would also have to do a lot of optional safeguarding in all the methods referencing the cards array (resetPageControl, calculateItemSize, handlePageControlValueChange). So I think it's more neat to default to an empty cards array in this instance. Let me know if you disagree :)